src/Components/Authentication/Desktop/LoginForm.tsx

```diff
@@ -26,11 +27,18 @@ export class LoginForm extends Component<FormProps, LoginFormState> {
     error: this.props.error,
   }
 
+  onSubmit = (values: InputValues, formikBag: FormikProps<InputValues>) => {
+    window.grecaptcha.ready(() => {
+      window.grecaptcha.execute(sd.RECAPTCHA_KEY, { action: "login_submit" })
+      this.props.handleSubmit(values, formikBag)
+    })
```

Comment:
It seems like maybe these grecaptcha calls can live inside of a util function: 

```js
export function showRecaptcha(action: string) {
  window.grecaptcha.ready(() => {
    window.grecaptcha.execute(sd.RECAPTCHA_KEY, { 
      action
    })
  })
}
```

And then functionality tests can be localized here, while simple assertions about whether `showRecaptcha` has been called take place elsewhere.